apicurito/deploy/cluster_role_binding.yaml

```diff
@@ -9,4 +9,4 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: apicurito
-    namespace: placeholder
+    namespace: ms-test
```

Comment:
make sure, revert this change

Reply:
dont need to review just yet. 
things are in WIP.